spec.html

```diff
@@ -12039,6 +12039,8 @@ <h1>[[Get]] ( _P_, _Receiver_ )</h1>
           1. Assert: _targetModule_ is not *undefined*.
           1. If _binding_.[[BindingName]] is *"\*namespace\*"*, then
             1. Return ? GetModuleNamespace(_targetModule_).
+          1. If _binding_.[[BindingName]] is *"\*ambiguous\*"*, then
+            1. Return *undefined*.
```

Comment:
would it be worth exploring the web compatibility of making this throw?

Reply:
It's difficult to say, but it likely is a web compat issue at this point I'd guess. It could still be worth discussing further though. Another concern about throwing would be even just logging a namespace to the console would be enough to cause the error.